requirements.in

```diff
@@ -13,10 +13,10 @@ jsl
 jsonschema
 BigQuery-Python
 unidiff
-requests
```

Comment:
Not sure about this.
You're right that it's installed in base image.
But `requirements.[in|txt]` does not specify what to install into image, but what are the requirements of the project.
And `requests` is still a requirement even it's already been included in base image.